src/components/DataDimension/DraggingItem.js

```diff
@@ -0,0 +1,47 @@
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { DIMENSION_TYPE_DATA_ELEMENT } from '../../modules/dataTypes.js'
+import { getIcon } from '../../modules/dimensionListItem.js'
+import { TYPE_DATAELEMENT, TYPE_INPUT, TYPE_OPERATOR } from './constants.js'
+import styles from './styles/DraggingItem.style.js'
+
+const DraggingItem = ({ item }) => {
+    const displayLabel =
+        item.type === TYPE_INPUT ? item.value || item.label : item.label
+
+    return (
+        <>
+            <div
+                className={cx('dragging-item', {
+                    operator: item.type === TYPE_OPERATOR,
+                    number: item.type === TYPE_INPUT,
+                    dataelement: item.type === TYPE_DATAELEMENT,
```

Comment:
```suggestion
                    dataElement: item.type === TYPE_DATA_ELEMENT,
```

Reply:
actually, the key is a css class, so should be `kebab-case` and the condition could just use `DIMENSION_TYPE_DATA_ELEMENT` instead, so that makes for:
```suggestion
                    data-element: item.type === DIMENSION_TYPE_DATA_ELEMENT,
```
(possibly needs to wrap the id in quotes)